### ISSUE TITLE
segmentation fault 

### ISSUE THREAD

**Issue by Jaykef** [pytgcalls/pytgcalls#196]
Code runs well then suddenly stops:

PyTgCalls v1.0.5 powered by NTgCalls v1.0.0.dev24, Copyright (C) 2021-2023 Laky-64 <https://github.com/Laky-64>
Licensed under the terms of the GNU Lesser General Public License v3 or later (LGPLv3+)

zsh: segmentation fault  python3 video.py

**Comment by Laky-64** [comment]
Can you share more details? Like if is reproducible and etc

**Comment by Jaykef** [comment]
> Can you share more details? Like if is reproducible and etc

Yes I'm was just trying out example/simple_calls/example_simple.py
with pyrogram client (got a connection)
but when I enter "!play" command in the group chat, it crashes:
<img width="546" alt="error" src="https://github.com/pytgcalls/pytgcalls/assets/11355002/853b50f2-25a5-457c-9688-68e997474f6c">

**Comment by Laky-64** [comment]
It happens always or happened just one time?
